### ISSUE TITLE
transfer string to double got fatal error!

### ISSUE THREAD

**Issue by dawnFromDark** [EOSIO/eos#7339]
when i use this function `strtod` in my contract ,it will report a error like the following:
![image](https://user-images.githubusercontent.com/44474000/57601012-fc956b00-758d-11e9-86a9-d0ad1b747d4d.png)
i tested many times,and you guys can test on your enviroment.
my compiler is **eosio-cpp version 1.6.1**
and nodeos version is:**v1.7.3 master branch**
And the `strtod `function is standard function,right??????

**Comment by dawnFromDark** [comment]
when you execture your action which contain `strtod` ,it will get the error like that.

**Comment by taokayan** [comment]
I can reproduce this issue by calling "strtod" inside a smartcontract. looks like the implementation of "strtod" is buggy.

```
Error 3070002: Runtime Error Processing WASM
Error Details:
wabt execution failure (out of bounds memory access)
```

**Comment by taokayan** [comment]
The reason is that the current implementation of ```strtod``` uses some local variables of size larger than 8KB which results in stack underflow.

**Comment by dawnFromDark** [comment]
> The reason is that the current implementation of `strtod` uses some local variables of size larger than 8KB which results in stack underflow.
so ,how should we to solve this,honey?

**Comment by taokayan** [comment]
you can implement your own strtod, or wait for the fix (in https://github.com/EOSIO/musl/pull/10) in the next version of cdt

**Comment by dawnFromDark** [comment]
> you can implement your own strtod, or wait for the fix (in [EOSIO/musl#10](https://github.com/EOSIO/musl/pull/10)) in the next version of cdt

ok, i will wait that, thank you.
